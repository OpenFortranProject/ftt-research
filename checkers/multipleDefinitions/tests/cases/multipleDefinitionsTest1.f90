program test

  integer :: x, y


  print *, x
  print *, y
end program
