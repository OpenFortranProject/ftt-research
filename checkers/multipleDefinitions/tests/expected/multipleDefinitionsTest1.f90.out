Warning: environment variable COMPASS_PARAMETERS is not set, trying to use compass_parameters of the current path
 Running Prerequisite SgProject
Multiple Definitions: /home/dagit/research/checkers/multipleDefinitions/tests/cases/multipleDefinitionsTest1.f90:3.13-16: Multiple definitions in single declaration found: 'x, y'
